openassessment/locale/config.yaml

```diff
@@ -74,11 +74,19 @@ dummy_locales:
 
 # Directories we don't search for strings.
 ignore_dirs:
```

Comment:
That's very strange. I wonder what this configuration is there for then! It makes me think that we should probably switch to i18n-tool instead of trying to emulate it...